.gitignore

```diff
@@ -37,7 +37,6 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
-/typings
```

Comment:
Not sure why this was ignored, so calling attention to it here. Don't see any problems after removing it, and by convention this is where we'd store types that aren't available from `@types`. If we do want to ignore this, I can just rename that directory